GetNextTurn2.php

```diff
@@ -103,7 +103,7 @@
     PlayerLoseHealth($otherP, GetHealth($otherP));
     include "WriteGamestate.php";
   }
-  if($lastUpdate != 0 && $lastUpdate <= intval($updateNumber)) {
+  if($lastUpdate != 0 && intval($updateNumber) <= $lastUpdate) {
```

Comment:
What happened for you to make this backwards now

Reply:
It was backwards before, hence why I was worried that the server was borked. But it turned out it's fine because it's broken for another reason that cancels out this brokenness :|